src/org/pcic/util/Utils.groovy

```diff
@@ -0,0 +1,70 @@
+package org.pcic.util
+
+import org.pcic.GitUtils
+import org.pcic.DockerUtils
+
+
+class Utils implements Serializable {
+    def steps
+
+    Utils(steps) {
+        this.steps = steps
+    }
+
+    /**
+     * This method takes in a list of keys and a map, and returns a map with
+     * updates values.
```

Comment:
Comments should state the purpose and result (why and what) much more than the method (how). This reads more like method than purpose.